### ISSUE TITLE
Comparing Different models

### ISSUE THREAD

**Issue by mohammadshahabuddin** [stevliu/self-conditioned-gan#6]
Hi,
Could you please tell me how you compared different models? Did you use the same learning rate, number of epochs, Number of decay epochs, image size, optimizer among all models? Also, did you collect test results using the final saved generator or did you use the best results testing all saved generators at different epochs?

**Comment by stevliu** [comment]
Hi,

The learning rate, total epochs, image size, and optimizer were the same across all models. We did not use any learning rate scheduling. And we collected test results based on the best model during training.

**Comment by mohammadshahabuddin** [comment]
Thank you for the information! These will be very helpful.

Thanks,
Shahab
